magento/cart2quote/2017-02-01.yaml

```diff
@@ -0,0 +1,13 @@
+
+title:     Remote Code Execution in Qquoteadv/controllers/DownloadController.php
+link:      https://cart2quote.zendesk.com/hc/en-us/articles/115000616303--FIXED-Security-Vulnerability-in-downloadCustomOptionAction
+cve:       ~
+branches:
+    4:
+        time:     2017-02-01 10:45:00
+        versions: ['>=4.1.6', '<5.0'
```

Comment:
The advisory is by the way talking about 4.2.0 and above. Not sure what is right.